core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -523,24 +523,12 @@ class UserService(protected val userInfo: UserInfo,
   def startBillingProjectCreation(createProjectRequest: CreateRawlsBillingProjectFullRequest): Future[Unit] = {
     for {
       _ <- validateV1CreateProjectRequest(createProjectRequest)
-      _ <- checkServicePerimeterAccess(createProjectRequest.servicePerimeter)
+      _ <- BillingProjectOrchestrator.checkServicePerimeterAccess(samDAO, createProjectRequest.servicePerimeter, userInfo)
```

Comment:
The V1 endpoints are still in use and rely on the service perimeter access checks. I've refactored to a method on the companion class so both code paths for reuse.

Reply:
@marctalbott Excellent, I did not know that existed, I'll look into it.